openbb_terminal/account/account_controller.py

```diff
@@ -259,15 +268,21 @@ def call_list(self, other_args: List[str]):
                     size=ns_parser.size,
                 )
                 df, page, pages = get_personal_routines_info(response)
-                if not df.empty:
-                    self.REMOTE_CHOICES += list(df["name"])
-                    self.update_runtime_choices()
-                    display_personal_routines(df, page, pages)
-                else:
-                    console.print("[red]No routines found.[/red]")
+
+                if ns_parser.type == "personal":
+                    if not df.empty:
+                        console.print("[info]Displaying personal routines.[/info]")
```

Comment:
I'd remove this print here and the similar one for default routines, either the table pops up or is printed in the console if `USE_INTERACTIVE_DF='False'`

Reply:
I removed the print to let the table pop up and show the results.